routes.js

```diff
@@ -0,0 +1,5 @@
+page('/', homeController.index);
+page('/data', dataController.index);
+page('/about', aboutController.index);
+// page('/data/:id', questionController.index);
```

Comment:
I'm already seeing some commented code and console logs. Be sure you take the time to fit and finish your code base when pushing a final copy for review/submission. 

If you want to retain those comments and debugging tools, put that code on a branch and then freshen up your final copy before merging to master.